shipyard/src/main/java/org/jclouds/shipyard/domain/servicekeys/CreateServiceKey.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jclouds.shipyard.domain.servicekeys;
+
+import org.jclouds.json.SerializedNames;
+
+import com.google.auto.value.AutoValue;
+
+@AutoValue
+public abstract class CreateServiceKey {
+   
+   public abstract String description();
+      
+   CreateServiceKey() {
+   }
+
+   @SerializedNames({ "description" })
+   public static CreateServiceKey create(String description) {
+      if (description == null) description = "";
```

Comment:
Do we really want to do this instead of declaring the field `@Nullable`? What is the purpose of this?